eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -523,5 +541,4 @@ object Peer {
   case class RelayOnionMessage(messageId: ByteVector32, msg: OnionMessage, replyTo_opt: Option[typed.ActorRef[Status]])
 
   case class RelayUnknownMessage(unknownMessage: UnknownMessage)
-  // @formatter:on
```

Comment:
nit: why did you remove this? When we create a `formatter:off` block, we always end it with a `formatter:on`

Reply:
I'm not sure how I missed that, fixed in [a00f448](https://github.com/ACINQ/eclair/pull/2601/commits/a00f448df3d45b3ab26952948a3c47f6cfeec9c0)